### ISSUE TITLE
.NET Standard nuget package pollutes references for non-.NET Standard/Core projects

### ISSUE THREAD

**Issue by lassevk** [lassevk/DiffLib#18]
When adding a reference to DiffLib from a normal .NET Framework project, a ton of .NET Standard nuget packages are added as well.

2 things needs to be done/looked into:

1. Can I specifically specify .NET Standard assemblies/packages I need
2. Create targets in the nuget package for .NET 3.5, 4.5, 4.6, and 4.7

**Comment by lassevk** [comment]
Will create a new issue for looking into bulletpoint 1.

Issue #20 has been created for this.

**Comment by drewnoakes** [comment]
I hadn't upgraded because of this issue, so am interested in this fix. What was the resolution in the end? Is there a release that includes this? Thanks!

**Comment by lassevk** [comment]
Hi, the latest version available on nuget already has this fix. If you install version 2017.7.26.1241 or later you will get specific assemblies for .NET 3.5, 4.5, 4.6 and 4.7, everything else will probably for now use the .NET Standard 1.0 version.

The fix was actually very simple, much simpler than I thought. In the .csproj file for DiffLib, which is also a nuspec file for .NET Standard class libraries, it said something like:

    <TargetFramework>netstandard1.0</TargetFramework>

The fix was simply to do this:

    <TargetFrameworks>netstandard1.0;net35;net45;net46;net47</TargetFrameworks>

* Make the `TargetFramework` tag plural, into `TargetFrameworks`
* Add the other target framework monikers
* Rebuild

The actual fix can be found [here](https://github.com/lassevk/DiffLib/commit/c7b7dd634790bf2629aa70721ffc6f7b8ac4f587#diff-ff7887af88808df2aee6ce57290ada1cL4).

I also had to persuade MSBuild to do it by issuing a clean, rebuild and then build command separately as it was genuinely confused about the whole issue but once I issued 3 calls to MSBuild the nupkg file was produced.

So the package is already on nuget.org, here: [DiffLib on nuget.org](https://www.nuget.org/packages/DiffLib/)

Sorry about the long time it took for me to fix this, I thought the fix involved a lot more and had a whole git branch where I experimented with various ways to create the nuget package that I just couldn't get to work. Turns out smarter heads than me had already solved it. :)
